package.json

```diff
@@ -14,8 +14,8 @@
     "test": "jest e2e-tests"
   },
   "dependencies": {
-    "css-loader": "^5.2.6",
-    "css-minimizer-webpack-plugin": "^3.0.1",
+    "css-loader": "^6.7.1",
```

Comment:
Are these upgrades are fully tested?

Reply:
Not checked thoroughly the whole tool at local but it was working fine wrt the pages i visited. 
However, gonna test whole website.